### PR TITLE
Renamed content-bundle to page-bundle

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | yes - a lot
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | sulu/sulu-minimal#146
| License | MIT
| Documentation PR | sulu/sulu-docs#417

#### What's in this PR?

This PR renames SuluContentBundle to SuluWebspaceBundle.

#### Why?

This will allow us to create an own SuluContentBundle for storing content data.

#### BC Breaks/Deprecations

Following things have changed:

* The bundle name changed from `SuluContentBundle` to `SuluPageBundle`
* All services or parameters prefixed with `sulu_content` will now use the prefix `sulu_page`
* The config tree for `sulu_content` was renamed to `sulu_page`
* All the Classes from the namespace `Sulu\Bundle\ContentBundle` moved to `Sulu\Bundle\PageBundle`

#### To Do

- [x] Create a documentation PR
- [x] Add breaking changes to UPGRADE.md

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -258,9 +258,9 @@ sulu_admin:
                 adapter: 'column_list'
                 displayProperties: 'title'
                 icon: 'su-document'
-                label: 'sulu_content.selection_label'
```
Comment (wachterjohannes) [inline UPGRADE.md]: revert upgrade file

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php [hunk 2]
```diff
@@ -72,8 +72,8 @@ public function __construct(
     public function getFunctions()
     {
         return [
-            new \Twig_SimpleFunction('sulu_content_load', [$this, 'load']),
-            new \Twig_SimpleFunction('sulu_content_load_parent', [$this, 'loadParent']),
+            new \Twig_SimpleFunction('sulu_webspace_load', [$this, 'load']),
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php]: revert

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php [hunk 3]
```diff
@@ -72,8 +72,8 @@ public function __construct(
...
+            new \Twig_SimpleFunction('sulu_webspace_load_parent', [$this, 'loadParent']),
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php]: revert

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php [hunk 4]
```diff
@@ -50,8 +50,8 @@ public function __construct(
     public function getFunctions()
     {
         return [
-            new \Twig_SimpleFunction('sulu_content_path', [$this, 'getContentPath']),
-            new \Twig_SimpleFunction('sulu_content_root_path', [$this, 'getContentRootPath']),
+            new \Twig_SimpleFunction('sulu_webspace_path', [$this, 'getContentPath']),
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: revert

File: src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php [hunk 5]
```diff
@@ -50,8 +50,8 @@ public function __construct(
...
+            new \Twig_SimpleFunction('sulu_webspace_root_path', [$this, 'getContentRootPath']),
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php]: revert

File: CHANGELOG.md [hunk 6]
```diff
@@ -66,7 +66,7 @@ CHANGELOG for Sulu
 
 * 1.6.16 (2018-03-19)
     * ENHANCEMENT #3849 [AdminBundle]             Add Twig main blocks to be able to use inheritance
-    * HOTFIX      #3848 [ContentBundle]           Added check for unknown-document for "sulu_content_load"
```
Comment (wachterjohannes) [inline CHANGELOG.md]: revert this file

File: package.json [hunk 7]
```diff
@@ -14,7 +14,7 @@
     },
     "dependencies": {
         "sulu-admin-bundle": "file:src/Sulu/Bundle/AdminBundle/Resources/js",
-        "sulu-content-bundle": "file:src/Sulu/Bundle/ContentBundle/Resources/js",
+        "sulu-webspace-bundle": "file:src/Sulu/Bundle/WebspaceBundle/Resources/js",
```
Comment (danrot) [inline package.json]: order alphabethically

File: UPGRADE.md [hunk 8]
```diff
@@ -2,6 +2,15 @@
 
 ## dev-develop
 
+### Renamed ContentBundle to WebspaceBundle
+
+Following things have changed:
+
+* The bundle name changed from `SuluContentBundle` to `SuluWebspaceBundle`
+* All services or parameters prefixed with `sulu_content` will now use the prefix `sulu_webspace`
```
Comment (danrot) [inline UPGRADE.md]: Should that also be the case for all the twig functions?

Comment (danrot) [reply UPGRADE.md]: When thinking about it... That would mean that it results in `sulu_webspace_load` to load a page :see_no_evil: Maybe `sulu_page` would be really the better prefix?

Comment (wachterjohannes) [reply UPGRADE.md]: will be done in an own PR

**Pull request merged**